### PR TITLE
Jwoo/ios vc 2 view

### PR DESCRIPTION
**Pull request by nesalang**
fixes issue 1241

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+#import "ImageSet.h"
+#import "ACRUILabel.h"
+
+using namespace AdaptiveCards;
+
+@implementation ACRView
+{
+    ACOAdaptiveCard *_adaptiveCard;
+    ACOHostConfig *_hostConfig;
+    CGRect _guideFrame;
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: >CGRect _guideFrame; [](start = 4, length = 19)

Is this different from self.frame?

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRFactSetRenderer.mm [hunk 2]
```diff
@@ -54,7 +53,7 @@ - (UILabel *)buildLabel:(NSString *)text
     return lab;
 }
 - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
-rootViewController:(UIViewController *)vc
+rootView:(ACRView *)rootView
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRFactSetRenderer.mm]: Fix indentation in a lot of places.

**Comment by bviglietta** [comment]
> 

You don't really need to define this method. It doesn't do anything that wouldn't happen by default.

---
Refers to: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderResult.mm:21 in 28fe3ba. [](commit_id = 28fe3bac6dfc88fc325222f1f739e9d4d81450ef, deletion_comment = False)

File: source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m [hunk 3]
```diff
@@ -154,11 +154,9 @@ - (void)update:(NSString *) jsonStr
     ACRRenderResult *renderResult;
     ACOHostConfigParseResult *hostconfigParseResult = [ACOHostConfig fromJson:self.hostconfig];
     ACOAdaptiveCardParseResult *cardParseResult = [ACOAdaptiveCard fromJson:jsonStr];
-    if(cardParseResult.isValid)
-    {
-        renderResult = [ACRRenderer render:cardParseResult.card
-                                    config:hostconfigParseResult.config
-                                     frame:CGRectMake(0, 0, 500, 0)];
+    if(cardParseResult.isValid){
+        //renderResult = [ACRRenderer renderAsViewController:cardParseResult.card config:hostconfigParseResult.config frame:CGRectMake(0, 0, 500, 0) delegate:self];
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m]: >//renderResult = [](start = 8, length = 16)

Remove commented-out code.

**Comment by nesalang** [comment]
> 

you're right; thank you

---
In reply to: [373564202](https://github.com/Microsoft/AdaptiveCards/pull/1242#issuecomment-373564202) [](ancestors = 373564202)

---
Refers to: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderResult.mm:21 in 28fe3ba. [](commit_id = 28fe3bac6dfc88fc325222f1f739e9d4d81450ef, deletion_comment = False)

**Pull request merged**